packages/@okta/i18n/src/properties/login.properties

```diff
@@ -651,6 +651,9 @@ registration.default.callbackhook.error=We could not process your registration a
 registration.error.userName.invalidEmail=Invalid email address
 registration.error.password.passwordRequirementsNotMet=Password requirements were not met
 registration.error.userName.notUniqueWithinOrg=An account with that email already exists
+registration.error.notUniqueWithinOrg = A user with this {0} already exists
```

Comment:
This will produce invalid translations, as the argument **also** needs to be translated.

We need to make sure we use the entire message (`A user with this email already exists`) as a part of multiple i18n key entries.

Reply:
That makes sense @jmelberg-okta. 
But I am guessing the params object indicates that there is string interpolation. For the following example, the value in the param could be anything, not just "Email". So the fix here is to have the server respond with a more specific key?

```
 {
    "message": "Provided value for property 'Email' does not match required pattern",
    "i18n": {
      "key": "registration.error.doesNotMatchPattern",
      "params": [
        "Email"
      ]
    },
    "class": "ERROR"
  }
  ```